### PR TITLE
Restore proptest for incrementally collecting funding fees

### PR DESCRIPTION
**Pull request by klochowicz**
Check that we're rounding up when collecting the funding fees pro rata every
hour.

### CODE REVIEW

File: daemon/src/model/cfd.rs [hunk 1]
```diff
@@ -2272,6 +2272,31 @@ mod tests {
...
+            let fee_account = FeeAccount::new(Position::Long, Role::Taker);
+
+            let fee_account_whole_interval = fee_account.add_funding_fee(funding_fee_for_whole_interval);
+            let fee_account_one_hour = fee_account.add_funding_fee(funding_fee_for_one_hour);
+
+            let total_balance_when_collected_hourly = fee_account_one_hour.balance().checked_mul(SETTLEMENT_INTERVAL.whole_hours()).unwrap();
+            let total_balance_when_collected_for_whole_interval = fee_account_whole_interval.balance();
+
+            prop_assert!(
+                total_balance_when_collected_hourly >= total_balance_when_collected_for_whole_interval,
```
Comment (thomaseizinger) [inline daemon/src/model/cfd.rs:2295]: Better to subtract values from each other and assert that the diff is smaller than some small number.

**Review by thomaseizinger — CHANGES_REQUESTED** [review]

**Review by thomaseizinger — APPROVED** [review]

File: daemon/src/model/cfd.rs (hunk 1 above)
Comment (klochowicz) [reply daemon/src/model/cfd.rs:2295]: I derived some value by tried and error (if we'd like to make it tighter, we can always adjust the implementation). 30 sat seems good for now.

**Comment by klochowicz** [comment]
bors retry

**Comment by thomaseizinger** [comment]
> bors retry

Don't think a clippy error will be solved be retrying :)

**Pull request merged**